app/javascript/gobierto_visualizations/webapp/visualizations/treeMapNested.vue

```diff
@@ -172,10 +178,11 @@ export default {
       d3.select('.treemap-container')
         .remove()
         .exit();
-      d3.select('.nav-breadcrumbs')
+      d3.select('.treemap-nested-sidebar-nav-breadcumb')
         .remove()
         .exit();
 
+      const colors = createScaleColors(this.arrayValuesContractTypes.length, this.arrayValuesContractTypes);
```

Comment:
What if `this.arrayValuesContractTypes.length` is longer than the `colorsGobiertoExtend` palette in _createScaleColors_?